subsys/lorawan/nvm/Kconfig

```diff
@@ -0,0 +1,22 @@
+# LoRaWAN Non Volatile Memory configuration options
+
+# Copyright (c) 2021 Intellinium <giuliano.franchetto@intellinium.com>
+# SPDX-License-Identifier: Apache-2.0
+
+choice LORAWAN_NVM
+	bool "LoRaWAN NVM backend"
+	default LORAWAN_NVM_SETTINGS if SETTINGS
+	default LORAWAN_NVM_NONE
+
+config LORAWAN_NVM_NONE
```

Comment:
I think we can get rid of this option.

Reply:
What would happen if a user choose to use LoRaWAN without ```CONFIG_SETTINGS``` enabled ?
If we want to make LoRaWAN to only work if at least settings is enabled, should we ```select SETTINGS``` when enabling LoRaWAN ?

Reply:
Definitely not, I for example will need to write a custom backend for this as we don't use the settings API

Reply:
> Definitely not, I for example will need to write a custom backend for this as we don't use the settings API

Okay, in that case it makes sense!